src/styles/cards/content.scss

```diff
@@ -0,0 +1,12 @@
+$border-color: #f3f3f3;
+$background-color: #fff;
+$button-color: #00acdc;
+$main-color-font: #060b35;
+$second-color-font: #616070;
```

Comment:
move to the `variables.scss`, and move the `variables.scss` file out of `header` folder (this folder is redundant)